### DISCUSSION TITLE
Can I run laravel horizon on Plesk?

### DISCUSSION THREAD

**Discussion by spacedog4** [laravel/framework#33538] [General]
I'm installing laravel horizon on plesk, but plesk don't recommend instaling third-party services, in this case I'm worried about Supervisor, can I run laravel horizon thought plesk schedule (cron jobs) or something like that?

**Comment by GrahamCampbell** [comment]
I wouldn't recommend running Laravel on low quality shared hosting solutions like that, never mind horizon.

**Reply by spacedog4** [reply]
I use plesk with digital ocean (5$ dopplet), I have 4 websites hosting on it, with no problem, I didn't undestand your point
